extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -419,17 +419,17 @@ export class AmpIframe extends AMP.BaseElement {
     // Calculate new width and height of the container to include the padding.
     // If padding is negative, just use the requested width and height directly.
     let newHeight, newWidth;
-    if (height !== undefined) {
+    height = parseInt(height, 10);
+    if (!isNaN(height)) {
       newHeight = Math.max(
-        (this.element./*OK*/offsetHeight - this.iframe_./*OK*/offsetHeight)
-            + height,
-        height);
+          height + (this.element.offsetHeight - this.iframe_.offsetHeight),
```

Comment:
OH, forgot to add back.

:-( Chrome debugger will not inline variable value if there is the /*OK*/